.github/workflows/build-extensions.yml

```diff
@@ -56,33 +56,188 @@ jobs:
           - name: WASI-NN OpenVINO
             compiler: g++
             docker_tag: ubuntu-build-gcc
+            test_path: test/plugins/wasi_nn
+            test_bin: wasiNNTests
+            build_type: Debug
+          - name: WASI-NN OpenVINO
+            compiler: g++
+            docker_tag: ubuntu-build-gcc
+            test_path: test/plugins/wasi_nn
+            test_bin: wasiNNTests
             build_type: Release
```

Comment:
Do we need to check both Release type and Debug type?
I found that most of the WASI-NN part are not using Debug feature.